cli/cobra_test.go

```diff
@@ -32,28 +33,30 @@ func TestVisitAll(t *testing.T) {
 	assert.DeepEqual(t, expected, visited)
 }
 
-func TestCommandVendor(t *testing.T) {
+func TestCommandVendorAndVersion(t *testing.T) {
 	// Non plugin.
-	assert.Equal(t, commandVendor(&cobra.Command{Use: "test"}), "             ")
+	assert.Equal(t, commandVendorAndVersion(&cobra.Command{Use: "test"}), "")
 
 	// Plugins with various lengths of vendor.
 	for _, tc := range []struct {
 		vendor   string
+		version  string
 		expected string
 	}{
-		{vendor: "vendor", expected: "(vendor)     "},
-		{vendor: "vendor12345", expected: "(vendor12345)"},
-		{vendor: "vendor123456", expected: "(vendor1234…)"},
-		{vendor: "vendor1234567", expected: "(vendor1234…)"},
+		{vendor: "vendor", expected: "(vendor)"},
+		{vendor: "VeryLongVendorName", expected: "(VeryLongVendorName)"},
```

Comment:
Are the `veryLong` .. tests still relevant now that we no longer truncate?

Reply:
No. Removing.